pkg/minikube/registry/drvs/kvm2/kvm2.go

```diff
@@ -206,3 +221,26 @@ func isCurrentUserLibvirtGroupMember() (bool, error) {
 	}
 	return false, nil
 }
+
+// doesCurrentUserHaveLibvirtPolkitPermissions returns if the current user is allowed to perform the org.libvirt.unix.manage polkit action
+func doesCurrentUserHaveLibvirtPolkitPermissions() (bool, error) {
```

Comment:
can we rename this to something a little shorter? hasPolkitPermissions() is a much shorter but equally clear name